apache/zeppelin
# Fixing zeppelin-2281 by clearing output when data is available

### What is this PR for?
Fixing ZEPPELIN-2281 by clearing previous when new data is available


### What type of PR is it?
[Bug Fix]

### Todos

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-2281

### How should this be tested?
* Execute a paragraph and see the output
* Execute the same paragraph again but abort it before completion and see the output.Paragraph is clear and previous output is lost

### Screenshots (if appropriate)

### Questions:
* Does the licenses files need update?    NA
* Is there breaking changes for older versions? NA
* Does this needs documentation? NA
